docs/src/gui/qtvcp.adoc

```diff
@@ -262,195 +350,256 @@ net s32-in1        test_panel.doublescale_1-s   classicladder.0.s32in-00
 
 ----
 sudo apt-get install qttools5-dev-tools qttools5-dev libpython3-dev
 ----
 
 ////
 Luego necesita agregar la biblioteca de carga del módulo python.
-Qtvcp usa QT5 con python2. Esta combinación normalmente no está disponible
-desde repositorios. Puede compilarlo usted mismo, o hay versiones precompiladas
-disponible para sistemas comunes.
-En 'lib/python/qtvcp/designer' hay carpetas basadas en arquitecturas de sistema
-y luego la versión QT.
-Debe elegir la carpeta de arquitectura de la CPU y luego elegir la serie; 5.5,
-5.7 o 5.9 de Qt.
-Actualmente el estiramiento de Debian usa 5.7, Mint 12 usa 5.5, Mint 19 usa 5.9.
+QtVCP usa QT5 con python2. Esta combinación normalmente no está
+disponible desde repositorios. Puede compilarlo usted mismo, o hay
+versiones precompiladas disponible para sistemas comunes.
+En 'lib/python/qtvcp/designer' hay carpetas basadas en arquitecturas de
+sistema y luego la versión QT.
+Debe elegir la carpeta de arquitectura de la CPU y luego elegir la
+serie; 5.5, 5.7 o 5.9 de Qt.
+Actualmente el estiramiento de Debian usa 5.7, Mint 12 usa 5.5, Mint 19
+usa 5.9.
 En caso de duda, verifique la versión de QT5 en el sistema.
 
 Debe descomprimir el archivo y luego copiar esa versión adecuada de
 'libpyqt5_py2.so' a esta carpeta
 '/usr/lib/x86_64-linux-gnu/qt5/plugins/designer'.
 
-(x86_64-linux-gnu podría llamarse algo ligeramente diferente en diferentes
-sistemas)
+(x86_64-linux-gnu podría llamarse algo ligeramente diferente en
+diferentes sistemas)
 
-Necesitará privilegios de superusuario para copiar el archivo en la carpeta.
+Necesitará privilegios de superusuario para copiar el archivo en la
+carpeta.
 ////
 
-Then you must add a link to the `qtvcp_plugin.py` to the folder that designer will search.
+Then you must add a link to the `qtvcp_plugin.py` in one of the folders
+that Qt Designer will search into.
 
-In a RIP version of linuxcnc `qtvcp_plugin.py` will be in:
+In a _RIP_ version of LinuxCNC `qtvcp_plugin.py` will be:
 
   '~/LINUXCNC_PROJECT_NAME/lib/python/qtvcp/plugins/qtvcp_plugin.py'
 
-Package installed version should be:
+_Package installed_ version should be:
 
   'usr/lib/python2.7/qtvcp/plugins/qtvcp_plugin.py' or
   'usr/lib/python2.7/dist-packages/qtvcp/plugins/qtvcp_plugin.py'
 
 Make a link file to the above file and move it to one of the places
-Designer searches in:
+Qt Designer searches in.
 
 Designer searches in these two place for links (pick one):
 This can be:
 
   '/usr/lib/x86_64-linux-gnu/qt5/plugins/designer/python' or
   '~/.designer/plugins/python'
 
-You may need to add the plugins/python folders
+You may need to create the `plugins/python` folder.
 
 To start Designer:
 
-for a RIP installed:
-open a terminal, set the environment for linuxcnc with the command: '. scripts/rip-environment'
-then load designer with : 'designer -qt=5'
+* For a _RIP install_: +
+  Open a terminal, set the environment for LinuxCNC <1>, then load
+  designer <2> with :
```

Comment:
I am not ultimately confident that I would want to read "Qt" here again, maybe better as "then load the designer..."?